api/src/services/collections.ts

```diff
@@ -219,7 +220,7 @@ export class CollectionsService {
 		const nestedActionEvents: ActionEventParams[] = [];
 
 		try {
-			const collections = await this.knex.transaction(async (trx) => {
+			const collections = transaction(this.knex, async (trx) => {
```

Comment:
Do we not need to `await` the transaction here as before?
```suggestion
			const collections = await transaction(this.knex, async (trx) => {
```